source/features/remove-profile-header.js

```diff
@@ -0,0 +1,4 @@
+export default function () {
+	$('.ProfileCanopy-header .ProfileCanopy-avatar').appendTo('.ProfileCanopy-inner .AppContainer');
+	$('.ProfileCanopy-header').remove();
```

Comment:
Use `.hide()`

Reply:
Not my code, just moved from the other file